.github/workflows/helm_lint.yml

```diff
@@ -10,7 +10,8 @@ jobs:
         uses: actions/checkout@v3
         with:
             fetch-depth: 0
-
+      - name: Fetch history
+        run: git fetch --all
```

Comment:
Just curious, why are we fetching all history?

Can you not just remove `fetch-depth: 0` like done [here](https://github.com/cfpb/consumerfinance.gov/blob/main/.github/workflows/backend.yml#L40-L50)?